### PR TITLE
Create mhfa.com.au.xml

### PR DESCRIPTION
**Pull request by sunnz**

### CODE REVIEW

**Comment by pipboy96** [comment]
Just a suggestion, generally, having same ruleset name and filename makes things easier.

**Comment by sunnz** [comment]
> Just a suggestion, generally, having same ruleset name and filename makes finding the ruleset in the repo much easier.

Sure, ruleset name updated to be the same as the filename now. :)

File: src/chrome/content/rules/mhfa.com.au.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="mhfa.com.au">
+  <target host="mhfa.com.au" />
```
Comment (pipboy96) [inline src/chrome/content/rules/mhfa.com.au.xml]: Do not use spaces for indentation, use `\t` instead.

**Comment by sunnz** [comment]
Changed indentation from spaces to tabs

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**